hadoop-ozone/dist/src/main/smoketest/spnego/web.robot

```diff
@@ -41,7 +41,7 @@ Verify SPNEGO enabled URL
     [arguments]                      ${url}
     Run Keyword if      '${SECURITY_ENABLED}' == 'true'     Execute     kdestroy
     ${result} =         Execute                             curl --negotiate -u : -v -s -I ${url}
-    Should contain      ${result}       401 Unauthorized
+    Should contain      ${result}       401 Authentication required
```

Comment:
This is changed by the Jetty version upgrade in Hadoop 3.3.1 (HADOOP-17371 ) where Hadoop adds back the ability to specify a custom error message (the standard error message is "401 Unauthorized") for Jetty >= 9.4.21.

By the HTTP standard, applications are not supposed to depend on the error message (only error code) so I think this is fine.